experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -190,9 +190,9 @@ export class NodeSDK {
   /**
    * Once the SDK has been configured, call this method to construct SDK components and register them with the OpenTelemetry API.
    */
-  public async start(): Promise<void> {
+  public start(): void {
```

Comment:
Also, the README for the package should be fixed. It currently [has this part](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node#initialize-the-sdk):
```js
sdk.start().then(() => {
  // Resources have been detected and SDK is started
});
```

Which needs to be updated